save-api/src/main/kotlin/com/saveourtool/save/api/impl/DefaultSaveCloudClient.kt

```diff
@@ -162,7 +165,9 @@ internal class DefaultSaveCloudClient(
                         headers
                     )
                 })
-            ).getOrElse { error ->
+            ) {
+                this[CONTENT_LENGTH_CUSTOM] = file.fileSize().toString()
```

Comment:
@akuleshov7 Here's the 2nd change.

Reply:
missed that, sounds like we made not so many in 6 months 😆

Reply:
There's also the change in the way TS version is now reported if the version is a Git branch name.

Used to be just the branch name, like `master`. Now it also includes the short commit hash, like `master (deadbeef)`.